mlos_bench/mlos_bench/tests/services/local/local_exec_test.py

```diff
@@ -86,6 +128,12 @@ def test_run_script_read_csv(local_exec_service: LocalExecService) -> None:
         assert stderr.strip() == ""
 
         data = pandas.read_csv(path_join(temp_dir, "output.csv"))
+        if sys.platform == 'win32':
+            # Workaround for Python's subprocess module on Windows adding a
+            # space inbetween the col1,col2 arg and the redirect symbol which
+            # cmd poorly interprets as being part of the original string arg.
+            # Without this, we get "col2 " as the second column name.
+            data.rename(str.rstrip, axis='columns', inplace=True)
```

Comment:
Wonder if it's worth us doing this inside the `read_results_file` routine and then calling that here directly.
@motus thoughts?

Reply:
yes, that's a good idea! we can do it in a separate PR - let me finish the `read_telemetry_file` and then we'll do it in a more generic way